cmd/executor/x/main.go

```diff
@@ -0,0 +1,223 @@
+package main
+
```

Comment:
BTW I'd follow-up with your idea about wrapping such binaries + templates in dedicated plugins as it could be hard for someone to use `flux` etc. 🤔 Do we have an issue for it already?

Reply:
Yeah, I can take this and discuss inside our team 👍